documentapi/src/vespa/documentapi/messagebus/documentprotocol.h

```diff
@@ -301,7 +300,6 @@ class DocumentProtocol final : public mbus::IProtocol {
      *
      * @return The system state.
      */
-    SystemState &getSystemState() { return *_systemState; }
```

Comment:
Comment should also be removed to avoid confusion

Reply:
Fixed